userdocs/profiles.dev/docs/author-docs/kustomize-and-raw.md

```diff
@@ -34,6 +34,16 @@ spec:
     # ...
 ```
 
+:::tip
+When you add local artifacts (meaning those with manifests stored in the profile repository)
+to your profile, we recommend noting that you have done
+so in your Readme or other documentation. Users will have to provide additional flags
```

Comment:
```suggestion
so in your Readme, or other documentation, because users of such profiles will have to provide additional flags to use it.
```

Reply:
This would make the sentence a bit too long and harder to follow.

Reply:
hmm actually maybe i disagree with myself